pro/src/store/StoreProvider/StoreProvider.tsx

```diff
@@ -42,6 +44,12 @@ const StoreProvider = ({
 
     const getStoreInitialState = async () => {
       await Promise.all([initializeUser(), initializeFeatures()])
+      const isLocalStorageAvailable = localStorageAvailable()
+
+      if (isLocalStorageAvailable) {
```

Comment:
Il n'y a pas vraiment besoin de la variable
```suggestion
      if (localStorageAvailable()) {
```